### PR TITLE
Added cert-manager and https cert automatic issuing

### PR DESCRIPTION
**Pull request by dm3ch**
Add installation of cert-manager

/fix https://github.com/chainstack/graph-deployment/issues/11

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,7 +4,6 @@ repos:
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
-    -   id: check-yaml
```
Comment (CaJIbHuK) [inline .pre-commit-config.yaml:7]: why?

**Review by CaJIbHuK — CHANGES_REQUESTED** [review]
WDYT if we install certmanager using helm provider like it is done here?
https://github.com/chainstack/infrastructure/tree/master/tf-modules/helm/certmanager

I don't want to increase the complexity of the repository by nesting directories. Also the number of manual steps users need to go through is undesirable IMHO.

I understand that you don't want to duplicate certmanager installation in each tf module.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (dm3ch) [reply .pre-commit-config.yaml:7]: Cause it triggers on helmfile.yaml gotpl parts

Comment (dm3ch) [reply .pre-commit-config.yaml:7]: Hmm, it started works. Reverted that change

**Comment by dm3ch** [comment]
> WDYT if we install certmanager using helm provider like it is done here?
> https://github.com/chainstack/infrastructure/tree/master/tf-modules/helm/certmanager
> 
> I don't want to increase the complexity of the repository by nesting directories. Also the number of manual steps users need to go through is undesirable IMHO.
> 
> I understand that you don't want to duplicate certmanager installation in each tf module.

I decided not to add that to terraform cause:
1) It could be useful for baremetal clusters
2) It doesn't depends on any terraform resource outputs.

I also don't like idea adding more nested structures, but I'm going to merge on helmfile in future, so I think it would be simplified in 3rd milestone.

P.S. Also this helmfile is going to contain kube-dashboard and prometheus+grafana installation

**Review by CaJIbHuK — APPROVED** [review]

**Pull request merged**